lib/tpw/unique.go

```diff
@@ -0,0 +1,17 @@
+package tpw
+
+import (
+	"encoding/hex"
+
+	"github.com/google/uuid"
+)
+
+func NewUniqueID() (string, error) {
```

Comment:
You could just use `uuid.NewString()` ?

Reply:
Also, `uuid.NewString()` _panics_ if the UUID fails to create for some reason. I prefer returning an error so as to allow the calling code to decide how to handle the failure.

Reply:
Sounds good !